komorebi/src/window_manager.rs

```diff
@@ -1631,14 +1632,24 @@ impl WindowManager {
                 let mouse_follows_focus = self.mouse_follows_focus;
 
                 if let Ok(focused_workspace) = self.focused_workspace_mut() {
-                    if let Some(monocle) = focused_workspace.monocle_container() {
+                    if let Some(window) = focused_workspace.maximized_window() {
+                        window.focus(mouse_follows_focus)?;
+                        // (alex-ds13): @LGUG2Z Why was this being done below on the monocle?
```

Comment:
Let's merge this and see if I notice any difference; if it feels off I'll add another commit on top doing this if `mouse_follows_focus` is true